pypit/arutils.py

```diff
@@ -133,27 +133,66 @@ def calc_offset(raA, decA, raB, decB, distance=False):
         return delRA, delDEC
 
 
-def dummy_self(pypitdir=None):
+def dummy_fitsdict(nfile=10):
+    """
+    Parameters
+    ----------
+    nfile : int
+      Number of files to mimic
+
+    Returns
+    -------
+
+    """
+    fitsdict = {}
+    fitsdict['date'] = ['2015-01-23T00:54:17.04']*nfile
+    fitsdict['target'] = ['Dummy']*nfile
+    #
+    return fitsdict
+
+
+def dummy_self(pypitdir=None, inum=0, fitsdict=None):
```

Comment:
Should we also specify `nfile=10` here, as one of the arguments?

and then replace line 176 with `spect[key]['index'] = [[jj]*nfile]`
and replace line 181 with `fitsdict = dummy_fitsdict(nfile=nfile)`